bun/components/TodoList.svelte

```diff
@@ -0,0 +1,123 @@
+<script>
+  // Define a Todo type using JSDoc for documentation purposes
+  /**
+   * @typedef {Object} Todo
+   * @property {string} text
+   * @property {boolean} completed
+   */
+
+  // Initialize an empty array for todos
+  /** @type {Todo[]} */
+  let todos = [];
```

Comment:
```suggestion
<script lang="ts">
  type Todo = {
    text: string;
    completed: boolean;
  };
  let todos: Todo[] = [];
```
You *can* use typescript here.